### ISSUE TITLE
Access Score Calculation So Inaccurate as Not to Be Useful

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#359]
Case in point--look at how many surface problems there are in Hill Crest and yet it still receives a good accessibility score. It's a bit embarrassing that it's marked as 99 out of 100 on our accessibility scale.

![image](https://cloud.githubusercontent.com/assets/1621749/19493504/ecb3caa6-9548-11e6-9be9-e36dbd2c3e7e.png)

In the short term, I think we should emphasize that Access Score is in beta somewhere in the text. In the longer term:
- I think we should make our access score algorithm more transparent. We should literally have a link to a page that describes our calculation/algorithm for this
- We should obviously improve this formula (@manaswis is currently working on it)
- We need to add other elements like road grade and elevation changes along a street segment

**Comment by manaswis** [comment]
PR #361 Short-term solution:

<img width="1279" alt="access-score-beta" src="https://cloud.githubusercontent.com/assets/2873216/19500775/5a7f5f12-956e-11e6-8739-e2d1216c9e14.png">

**Comment by jonfroehlich** [comment]
This continues to be rather embarrassing to be honest. I think we need to discuss whether we want to take down Access Score until the load times are better and, just as importantly, the access score model is improved. Check out the sheer number of orange 'Surface Problem' labels in Hill Crest and yet, it is rated as 99 'Very Accessible.' We can and should do better.

![image](https://cloud.githubusercontent.com/assets/1621749/20114416/4a24c904-a5c2-11e6-83dc-6be6a68faf62.png)

**Comment by misaugstad** [comment]
do we want to take this down, leave it as-is, or make any updates to it this summer?

**Comment by jonfroehlich** [comment]
Is this now the results page? Can you send a URL for me to check it out?

**Comment by misaugstad** [comment]
No, this is at the /demo endpoint, it was actually using accessscore, unlike /results I believe. 

https://sidewalk-newberg-test.cs.washington.edu/demo

**Comment by jonfroehlich** [comment]
This is actually related to @johnsonkuang's summer project. We should have a discussion.

**Comment by jonfroehlich** [comment]
Tagging @crescendochu so she sees this.

**Comment by misaugstad** [comment]
Closing this as we're going to be removing the /demo page.
